widgets/odm-related-content-widget.php

```diff
@@ -52,40 +52,43 @@ public function widget( $args, $instance ) {
     $limit = isset($instance['limit']) ? $instance['limit'] : -1;
     $type = isset($instance['type']) ? $instance['type'] : 'dataset';
 		$template = isset($instance['template']) ? $instance['template'] : 'default';
+		$related_content = get_post_meta($post->ID,'related_content',true);
+		$data = json_decode($related_content,true);
+		
+		if ($data !== null):
```

Comment:
@Huyeng I basically moved the following to lines to the top of the file:

```
$related_content = get_post_meta($post->ID,'related_content',true);
$data = json_decode($related_content,true);
```

and added this if clause which checks if the JSON file was decoded succesfully. if it wasn't then does nothing